packages/perseus/src/prompt-types.ts

```diff
@@ -0,0 +1,32 @@
+export enum WidgetType {
+    RADIO = "RADIO",
+}
+
+export type BasicOption = {
```

Comment:
If this is just used internally, I don't think we need to export it. Same with the others that aren't used outside this file.

In fact, since you made a separate file for the util, I wonder if these types should just be in that file?

For context, we've had a long history in Perseus of having several _really close_ types without understanding why they all existed. So if we could quarantine these types as much as possible I think we might be better off.

Also I have a hunch @jeremywiebe might swoop in and suggest we use existing types for these derivative types, ie:

``` TSX
export type RadioPromptJSON = {
  hasNoneOfTheAbove: PerseusRadioWidgetOptions["hasNoneOfTheAbove"];
}
```

Ultimately I'll defer to Jeremy since he's smarter than me, but I'm the one who suggested that we keep these types completely decoupled from our regular types because I thought it would be beneficial for TUT to have full control of the _output_ type as long as we kept full control of the _input_ type. I could totally be misguided with this suggestion.